webapp/niapp.yaml

```diff
@@ -0,0 +1,416 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: SystemLink WebApp Service
+basePath: /niapp/v1
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  ApiKeyAuth:
+    type: apiKey
+    name: x-ni-api-key
+    in: header
+security:
+  - ApiKeyAuth: []
+schemes:
+  - https
+paths:
+  '/webapps/{id}':
+    get:
+      tags:
+        - metadata
+      summary: Get webapp metadata by Id
+      description: Get webapp metadata by Id
+      operationId: get-webapp
+      parameters:
+        - $ref: '#/parameters/WebAppId'
+      responses:
+        200:
+          $ref: '#/responses/GetWebAppResponse'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+    put:
+      tags:
+        - metadata
+      summary: Update existing webapp metadata
```

Comment:
Nitpick, feel free to ignore: Usually the description is more specific than the summary, so it's weird that the summary is wordier than the description. They should probably either be the same, or the longer one should be in the description slot.